Dockerfile.garage-push

```diff
@@ -0,0 +1,10 @@
+FROM advancedtelematic/aktualizr
+LABEL Description="garage-push"
+
+ADD . src
+
+RUN apt-get install libsqlite3-dev
```

Comment:
I think you need apt-get update && apt-get install libsqlite3-dev here.  The reason is that if the advancedtelematic/aktualizr image is old then the apt metadata cache will be old, and the apt-get install will fail because it will try and download a package that is now obsolete.

The two commands need to be on one line, because Docker caches intermediate results at the line level.